data/announcements/scheduled.yaml

```diff
@@ -71,4 +71,26 @@ announcements:
     <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/">KubeCon + CloudNativeCon EU 2021</a> <em>virtual</em>.
   message: |
     4 days of incredible opportunities to collaborate, learn + share with the entire community!<br />
-    May 4 - May 7, 2021.
\ No newline at end of file
+    May 4 - May 7, 2021.
+
+- name: Kubecon 2021 NA
+  startTime: 2021-11-10T00:00:00
+  endTime: 2021-10-16T00:00:00
+  style: "background: linear-gradient(90deg, rgb(7, 132, 111) 0%, rgb(54, 214, 183) 100%)" 
+  title: |
+    <img src="https://events.linuxfoundation.org/wp-content/uploads/2020/11/KubeCon_NA_2021_web_web-logo-white-1.svg" style="float: right; height: 80px;" />
+    <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/"> KubeCon + CloudNativeCon North America 2021</a> <em>Los Angeles, California + Virtual</em>.
+  message: |
+    4 days of incredible opportunites to collaborate, learn + share with entire community!<br />
+    October 11 - 15, 2021.
+
+- name: Kubecon 2021 CHINA
+  startTime: 2021-09-12T00:00:00
```

Comment:
is this really intended to start ~3 months before the event?

Reply:
I would start the announcement around 10 days before the event.

Reply:
Ahhh I actually made a mistake with the format, I thought startTime & endTime was supposed to be the start time end time of the event

Reply:
Assigning @sftim since he is already engaged on this.
/assign @sftim